### ISSUE TITLE
Pink color

### ISSUE THREAD

**Issue by MangaD** [EliverLara/Ant-Dracula#22]
Hello, this theme is great but I don't fancy the pink color. Is there a way to easily change this?

**Comment by EliverLara** [comment]
Hi, thanks for the feedback! you can check how to change it [here](https://github.com/EliverLara/Ant/issues/27), What color do you think would be better?

**Comment by MangaD** [comment]
Hello @EliverLara, thank you very much for the feedback!

I have followed the instructions, so changed the `selected_bg_color` variable (for gtk < 3.20 in my case). It did change many things but many things still have the pink color, for example the XFCE workspace switcher, the HexChat textbox, the XFCE logout screen... Any help?

After some testing, I found the color #ff5e5e to be a good alternative (although blue would also be fine with me). Thank you!
